downloader/src/main/kotlin/VcsHost.kt

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright (C) 2019 Bosch Software Innovations GmbH
```

Comment:
This change strips the HERE copyright as the code existed before already - can you please add it?

Reply:
Good point, sorry, done.